src/libstd/primitive_docs.rs

```diff
@@ -363,10 +363,11 @@ mod prim_unit { }
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
 /// Working with raw pointers in Rust is uncommon, typically limited to a few patterns.
-/// Raw pointers can be unaligned or null when unused. However, when a raw pointer is used to
-/// load/store data from/to memory, they must be non-null and aligned.
-/// Storing through a raw pointer (`*ptr = data`) calls `drop` on the old value, so
-/// [`write`] must be used if memory is not already initialized.
+/// Raw pointers can be unaligned or [`null`] when unused. However, when a raw pointer is
+/// dereferenced (using the `*` operator), it must be non-null and aligned.
+/// Storing through a raw pointer using `*ptr = data` calls `drop` on the old value, so
+/// [`write`] must be used if memory is not already initialized---otherwise `drop`
+/// would be called on the uninitialized memory.
```

Comment:
If `T` is `Copy` and also uninitialized, do you need to call `p.write(t)` in that case? Since there's no `Drop` code that will run.

Reply:
I updated this a bit, does that look good to you?

Reply:
solid!